DCF_Demo.md

```diff
@@ -0,0 +1,513 @@
+# DCF Invoke Device Method Using Azure CLI
+
+## Open up an instance of Azure CLI 
+You can use the Azure Portal or an instance on Powershell/bash/other local environments
```

Comment:
let's remove PS for now bc it does not follow the same format

Reply:
Why not keep it and add a note to replace \" by \`" in PS?

Reply:
I think we should keep it simple. too many approaches will only make it more confusing than it needs to be

Reply:
I added a click to expand section so people can just open the one they want if that works, I personally don't have azcli installed on powershell and used the portal for convenience